src/mw-ui-components/directives/mw_option_group.js

```diff
@@ -6,13 +6,18 @@ angular.module('mwUI.UiComponents')
         title: '@',
         description: '@',
         icon: '@',
-        mwDisabled: '='
+        mwDisabled: '=',
+        badges: '='
```

Comment:
It is optional. The only condition it that it has to be an array of strings as soon as it is set